staging/src/k8s.io/apiserver/pkg/storage/value/metrics.go

```diff
@@ -51,7 +51,7 @@ var (
 			Buckets:        metrics.ExponentialBuckets(5e-6, 2, 25),
 			StabilityLevel: metrics.ALPHA,
 		},
-		[]string{"transformation_type"},
+		[]string{"transformation_type", "transformer_prefix", "status"},
```

Comment:
This requires a mention in the release notes?

Reply:
Added